runtime/src/main/java/org/corfudb/runtime/collections/CorfuStoreShim.java

```diff
@@ -97,7 +97,7 @@ Table<Queue.CorfuGuidMsg, V, Queue.CorfuQueueMetadataMsg> openQueue(@Nonnull Str
      * @param tableName Table name.
      */
     public void deleteTable(String namespace, String tableName) {
```

Comment:
I think anyone using deleteTable() is probably looking for dropTable() so we might as well remove this method rather than just make it clear the table in my opinion. It will reduce our API footprint

Reply:
I think it would be meaningful to keep the clearTable because it sounds like a very common API. And just like any SQL database, usually both clear and drop operations are supported. Whether clients really want clear or drop in their code requires a closer look into each usage, so won't it be better to make the changes at each usage?

Reply:
sure, sounds reasonable - we can keep it then.